.github/workflows/release.yml

```
@@ -63,15 +63,14 @@ jobs:
       - uses: actions/checkout@v4
-      - uses: softprops/action-gh-release@v1
-        with:
-          generate_release_notes: true
-          files: target/release/${{ env.REPO_NAME }}
       - uses: actions/download-artifact@v4
         with:
           name: ${{ env.REPO_NAME }}
-      - run: gh release upload "${{ github.ref_name }}" ${{ env.REPO_NAME }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)